### PR TITLE
Fix CSS of Headline Blocks (#32)

### PR DESCRIPTION
**Pull request by iRohitSingh**
View : 

Screenshot 2023-06-22 at 14 53 33

Edit : 

Screenshot 2023-06-22 at 14 54 11

### CODE REVIEW

File: src/theme/blocks/_heading.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.block.heading {
+  margin-bottom: 0 !important;
+  .heading-wrapper {
+    h2 {
+      padding-top: 80px;
```
Comment (sneridagh) [inline src/theme/blocks/_heading.scss:5]: We have mixins for this now in `variables`. What was the issue? I thought this was fixed.

Comment (iRohitSingh) [reply src/theme/blocks/_heading.scss:5]: Edit mode is not similiar to view mode.
screenshot
Screenshot 2023-06-23 at 14 24 05

Comment (iRohitSingh) [reply src/theme/blocks/_heading.scss:5]: I have fix the problem using padding making similiar to dlr.

**Comment by sneridagh** [comment]
@iRohitSingh problem is that we are using a completely different approach with the light theme and the auto grouping.
The fix you are searching only have to work on edit, then only apply there. If you take a closer look, in view mode, the spacing is broken now (your screenshot), because you add the padding above the existing margins.

**Comment by sneridagh** [comment]
@iRohitSingh no worries, let me take care of it!

**Comment by sneridagh** [comment]
@iRohitSingh I just tried and it's fine as it is, true that it does not forces the margins in the same way as in DLR, but I feel that it's better like this.

image

image

**Pull request closed without merging**